keras/random/random.py

```diff
@@ -231,3 +231,62 @@ def gamma(shape, alpha, dtype=None, seed=None):
             of `keras.random.SeedGenerator`.
     """
     return backend.random.gamma(shape, alpha=alpha, dtype=dtype, seed=seed)
+
+
+def binomial(shape, counts, probs, dtype=None, seed=None):
+    """Draw samples from a Binomial distribution.
+
+    The values are drawn from a binomial distribution with
+    specified count and probability of success parameters.
+
+    Args:
+        shape: The shape of the random values to generate.
+        counts: A number or array of numbers representing the
+            number of trials. It must be broadcastable with probs.
+        probs: A float or array of floats representing the
```

Comment:
probs -> probabilities (we avoid abbreviations as much as possible, and there's no standard API to follow here since each framework uses a different name)